github/repos.go

```diff
@@ -557,6 +557,12 @@ type RequiredStatusChecks struct {
 	Contexts []string `json:"contexts"`
 }
 
+// RequiredStatusChecksRequest represents a request to edit a protected branch's status checks.
+type RequiredStatusChecksRequest struct {
```

Comment:
Adding request object because `UpdateRequiredStatusChecks` is a `PATCH` function so fields can be omitted in requests.